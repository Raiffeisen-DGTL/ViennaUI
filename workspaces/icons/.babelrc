{
  "presets": ["@babel/env", "@babel/typescript", "@babel/react"],
  "env": {
    "esm": {
      "presets": [["@babel/env", { "modules": false }]]
    }
  }
}
