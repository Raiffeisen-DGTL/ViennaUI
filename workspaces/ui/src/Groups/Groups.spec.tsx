import React from 'react';
import { Trash } from 'vienna.icons';
import Button from '../Button/Button';
import Groups, { GroupsProps } from './Groups';

test('Groups', () => {
    const snap = snapshot.render(
        <Groups>
            <Button design='accent'>Make payment</Button>
            <>
                <Button design='outline'>Download statement</Button>
                <Button design='outline'>
                    <Trash size='xs' /> Delete statement
                </Button>
            </>
        </Groups>
    );

    expect(snap).toMatchSnapshot();
});

test('Groups w/ null', () => {
    const snap = snapshot.render(
        <Groups>
            {null}
            <Button design='accent'>Make payment</Button>
            {false}
            <Button design='outline'>
                <Trash size='xs' /> Delete statement
            </Button>
        </Groups>
    );

    expect(snap).toMatchSnapshot();
});

test('Groups w/ groups', () => {
    const snap = snapshot.render(
        <Groups>
            <Button design='accent'>Make payment</Button>
            <Groups>
                <Button design='outline'>Download statement</Button>
                <Button design='outline'>
                    <Trash size='xs' /> Delete statement
                </Button>
            </Groups>
        </Groups>
    );

    expect(snap).toMatchSnapshot();
});

test('Groups w/ design', () => {
    const designs: GroupsProps['design'][] = ['horizontal', 'vertical'];

    designs.forEach((design) => {
        const snap = snapshot.render(
            <Groups design={design}>
                <Button design='accent'>Make payment</Button>
                <Button design='outline'>Download statement</Button>
                <Button design='outline'>
                    <Trash size='xs' /> Delete statement
                </Button>
            </Groups>
        );

        expect(snap).toMatchSnapshot(design);
    });
});

test('Groups w/ size', () => {
    const sizes: GroupsProps['size'][] = ['xs', 's', 'm', 'l', 'xl'];

    sizes.forEach((size) => {
        const snap = snapshot.render(
            <Groups size={size}>
                <Button design='accent'>Make payment</Button>
                <Button design='outline'>Download statement</Button>
                <Button design='outline'>
                    <Trash size='xs' /> Delete statement
                </Button>
            </Groups>
        );

        expect(snap).toMatchSnapshot(size);
    });
});

test('Groups w/ height', () => {
    const snap = snapshot.render(
        <Groups height='full'>
            <Button design='accent'>Make payment</Button>
            <Groups>
                <Button design='outline'>Download statement</Button>
                <Button design='outline'>
                    <Trash size='xs' /> Delete statement
                </Button>
            </Groups>
        </Groups>
    );

    expect(snap).toMatchSnapshot();
});

test('Groups w/ alignItems', () => {
    const snap = snapshot.render(
        <Groups alignItems='stretch'>
            <Button design='accent'>Make payment</Button>
            <Button design='outline'>Download statement</Button>
            <Button design='outline'>
                <Trash size='xs' /> Delete statement
            </Button>
        </Groups>
    );

    expect(snap).toMatchSnapshot();
});

test('Groups w/ justifyContent', () => {
    const snap = snapshot.render(
        <Groups justifyContent='center'>
            <Button design='accent'>Make payment</Button>
            <Button design='outline'>Download statement</Button>
            <Button design='outline'>
                <Trash size='xs' /> Delete statement
            </Button>
        </Groups>
    );

    expect(snap).toMatchSnapshot();
});

test('Groups w/ any props', () => {
    const props = { className: 'className', id: 'Counter', title: 'title', onClick: () => 'Counter' };
    const component = (
        <Groups {...props}>
            <Button design='accent'>Make payment</Button>
            <Button design='outline'>Download statement</Button>
            <Button design='outline'>
                <Trash size='xs' /> Delete statement
            </Button>
        </Groups>
    );

    const snap = snapshot.render(component);
    expect(snap).toMatchSnapshot();

    const wrapper = snapshot.shallow(component);
    for (const [key, value] of Object.entries(props)) {
        expect(wrapper.props()[key]).toEqual(value);
    }
});

test('Groups w/ clean (React.Fragment remove)', () => {
    const snap = snapshot.render(
        <Groups clean>
            <Button>{'A'}</Button>

            {true}
            {false}
            {null}
            {undefined}

            {
                <>
                    <Button>{'B'}</Button>
                    {true}
                    <Button>{'C'}</Button>
                    {null}
                </>
            }

            {
                <>
                    <Button>{'D'}</Button>
                    {false}
                    <Button>{'E'}</Button>
                    {undefined}
                </>
            }

            {
                <>
                    <Button>{'F'}</Button>
                </>
            }

            {
                <>
                    {true}
                    {false}
                    {null}
                    {undefined}
                </>
            }

            <Button>{'J'}</Button>
        </Groups>
    );

    expect(snap).toMatchSnapshot();
});
