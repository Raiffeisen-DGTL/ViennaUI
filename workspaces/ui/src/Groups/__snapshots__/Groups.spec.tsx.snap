// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Groups 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 fGINAc"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Download statement
      </span>
    </button>
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ alignItems 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 gDfoPk"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Download statement
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ any props 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 fGINAc className"
  id="Counter"
  title="title"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Download statement
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ clean (React.Fragment remove) 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 fGINAc"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 dcKhhz"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        A
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 dcKhhz"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        B
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 dcKhhz"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        C
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 dcKhhz"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        D
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 dcKhhz"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        E
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 dcKhhz"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        F
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 dcKhhz"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        J
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ design: horizontal 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 fGINAc"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Download statement
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ design: vertical 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 iFNDcQ"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 bcXHUW"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 bcXHUW"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Download statement
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 bcXHUW"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ groups 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 fGINAc"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <div
      class="Groupsstyles__Box-sc-1dief2o-0 fGINAc"
    >
      <div
        class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
      >
        <button
          class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
          type="button"
        >
          <span
            class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
          >
            Download statement
          </span>
        </button>
      </div>
      <div
        class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
      >
        <button
          class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
          type="button"
        >
          <span
            class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
          >
            <svg
              fill="currentColor"
              height="12"
              viewBox="0 0 24 24"
              width="12"
              xmlns="http://www.w3.org/2000/svg"
            >
              <defs />
              <path
                d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
              />
            </svg>
          </span>
          <span
            class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
          >
             Delete statement
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Groups w/ height 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 hxvwrz"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <div
      class="Groupsstyles__Box-sc-1dief2o-0 fGINAc"
    >
      <div
        class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
      >
        <button
          class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
          type="button"
        >
          <span
            class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
          >
            Download statement
          </span>
        </button>
      </div>
      <div
        class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
      >
        <button
          class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
          type="button"
        >
          <span
            class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
          >
            <svg
              fill="currentColor"
              height="12"
              viewBox="0 0 24 24"
              width="12"
              xmlns="http://www.w3.org/2000/svg"
            >
              <defs />
              <path
                d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
              />
            </svg>
          </span>
          <span
            class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
          >
             Delete statement
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Groups w/ justifyContent 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 hDAsnh"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Download statement
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ null 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 fGINAc"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ size: l 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 eZgkfB"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 hoVngd"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 hoVngd"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Download statement
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 hoVngd"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ size: m 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 fGINAc"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Download statement
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 htkhjN"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ size: s 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 fBejJs"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 gOSTmR"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 gOSTmR"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Download statement
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 gOSTmR"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ size: xl 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 feKNWl"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 hTmBcZ"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 hTmBcZ"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Download statement
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 hTmBcZ"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;

exports[`Groups w/ size: xs 1`] = `
<div
  class="Groupsstyles__Box-sc-1dief2o-0 dJximv"
>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 cmOVpV"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 jolcIP"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Make payment
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 cmOVpV"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        Download statement
      </span>
    </button>
  </div>
  <div
    class="Groupsstyles__Item-sc-1dief2o-1 cmOVpV"
  >
    <button
      class="Buttonstyles__Box-qfvnbt-0 ebCIkw"
      type="button"
    >
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
        <svg
          fill="currentColor"
          height="12"
          viewBox="0 0 24 24"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <defs />
          <path
            d="M7 9v11h10V9h2v12a1 1 0 01-1 1H6a1 1 0 01-1-1V9h2zm4 0v9H9V9h2zm4 0v9h-2V9h2zm0-7a1 1 0 01.949.684l.771 2.315L20 5v2H4V5l3.279-.001.772-2.315A1 1 0 019 2h6zm-.72 2H9.72l-.333.999h5.225L14.279 4z"
          />
        </svg>
      </span>
      <span
        class="Buttonstyles__ContentWrapper-qfvnbt-1 kJiCTN"
      >
         Delete statement
      </span>
    </button>
  </div>
</div>
`;
